src/seedu/addressbook/parser/Parser.java

```diff
@@ -73,33 +74,37 @@ public Command parseCommand(String userInput) {
 
         switch (commandWord) {
 
-        case AddCommand.COMMAND_WORD:
-            return prepareAdd(arguments);
+            case SortCommand.COMMAND_WORD:
```

Comment:
Wrong indentation? Our coding standard requires 4 spaces (instead of tabs) for normal indentation and an indentation of 8 spaces for line wrapping (i.e. when breaking a long line into multiple lines).